client/me/form-base/with-form-base.jsx

```diff
@@ -8,96 +8,54 @@ import { connect } from 'react-redux';
 /**
  * Internal dependencies
  */
-import user from 'calypso/lib/user';
-import userSettings from 'calypso/lib/user-settings';
 import { errorNotice, successNotice } from 'calypso/state/notices/actions';
+import QueryUserSettings from 'calypso/components/data/query-user-settings';
```

Comment:
Nit: would be great to sort imports alphabetically since we're touching all of them here.

Reply:
Did you miss that comment, or you simply disagree 😉  Let me know if I should stop bugging you with those 😜

Reply:
Oops, I missed it. I'll add another commit :) sorry